.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: python
+python:
+    - "2.6"
```

Comment:
This list should match what is in the tox.ini file (you probably can do this entire list, but you may need to fine tune it.